### PR TITLE
ref: type src/sentry/api/bases/organization.py

### PR DESCRIPTION
**Pull request by asottile-sentry**

### CODE REVIEW

**Review by evanh — APPROVED** [review]

File: src/sentry/api/bases/organization.py [hunk 1]
```diff
@@ -336,7 +347,7 @@ def get_filter_params(
             raise ParseError(detail=f"Invalid date range: {e}")
 
         try:
-            projects = self.get_projects(request, organization, project_ids)
+            projects = self.get_projects(request, organization, project_ids=project_ids)
```
Comment (asottile-sentry) [inline src/sentry/api/bases/organization.py:350]: hmmm there are test failures pointing at this line of code being the culprit

pretty sure this code is just broken as the error before this fix is:

```
src/sentry/api/bases/organization.py:350: error: Argument 3 to "get_projects" of "OrganizationEndpoint" has incompatible type "Optional[Set[int]]"; expected "bool"  [arg-type]
```

and the function is:

```python
    def get_projects(
        self,
        request: HttpRequest,
        organization: Organization,
        force_global_perms: bool = False,
        include_all_accessible: bool = False,
        project_ids: set[int] | None = None,
        project_slugs: set[str] | None = None,
    ) -> list[Project]:
```

so today it's accidentally passing `project_ids` into `force_global_perms`

Comment (corps) [reply src/sentry/api/bases/organization.py:350]: I think, as weird as it sounds, it's possible that this is intentional.
Like, maybe the line is supposed to read `self.get_projects(request, organization, force_global_perms=project_ids > 0)`

Comment (wedamija) [reply src/sentry/api/bases/organization.py:350]: I don't think that's the case. When I looked at the blame, the `project_ids` param was added to `self.get_projects` in the same pr, which implies it's a bug. @wmak do you have any context here?

Comment (wmak) [reply src/sentry/api/bases/organization.py:350]: I don't think that's the case either, I think this is a bug. But I'm sorta surprised we never caught it before now

**Review by wmak — APPROVED** [review]
LGTM, but I wonder if we should split the project_id fix into a separate PR so we're not doing feature changes mixed into a typing change?

**Pull request merged**